app/src/main/java/com/ccc/chatapp/screens/signIn/SignInPresenter.kt

```diff
@@ -51,4 +49,4 @@ class SignInPresenterImpl(
             })
         mCompositeDisposable.add(disposable)
     }
-}
+}
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/onstonboy/3C_ChatApp_Training/blob/0229390d5df1c0379ca4044bebed2a3c66cd4802/app/src/main/java/com/ccc/chatapp/screens/signIn/SignInPresenter.kt#L52"></span>The file SignInPresenter.kt is not ending with a new line.</td>
    </tr>
  </tbody>
</table>